frontend/containers/DocumentListToolbar/DocumentListToolbar.jsx

```diff
@@ -264,13 +266,14 @@ class DocumentListToolbar extends Component {
 
     // We might want to change this when we allow a field to reference multiple
     // documents. For now, we just get the first selected document.
-    const selectedDocumentId = state.documents.selected[0]
-    const selectedDocument = documentsList.find(document => {
-      return document._id === selectedDocumentId
-    })
+    const selectedDocuments = state.documents.selected.map(documentId => {
+      return documentsList.find(document => {
+        return document._id === documentId
+      })
+    }).filter(Boolean)
```

Comment:
I personally like to use shorthand arrow functions for simple and short statements. IMO having a `.find()` inside a shorthand `.map()` is unnecessarily cryptic.